CONTRIBUTING.md

```diff
@@ -148,37 +198,55 @@ yarn test                      # A good idea to make sure tests run first
-(`View > Toggle Developer Tools`), hover over them, and press
-<kbd>Cmd</kbd> + <kbd>R</kbd> (macOS) or <kbd>Ctrl</kbd> + <kbd>R</kbd>
-(Windows & Linux).
+This will build the project and start the application in production mode.
 
-```
-yarn build-everything:watch # runs until you stop it, re-generating built assets on file changes
-# Once this command is waiting for changes, you will need to run in another terminal `yarn worker:utils && yarn worker:libsession` to fix the "exports undefined" error on start.
-# If you do change the sass while this command is running, it won't pick it up. You need to either run `yarn sass` or have `yarn sass:watch` running in a separate terminal.
+### Hot reloading
+
+More often than not, you'll need to restart the application regularly to see your changes, as there
+is no automatic restart mechanism for the entire application.
+
+You can keep the developer tools open (`View > Toggle Developer Tools`) and press <kbd>Cmd</kbd> + <kbd>R</kbd> (macOS) or <kbd>Ctrl</kbd> + <kbd>R</kbd> (Windows & Linux) to reload the application frontend.
+
+```shell
+# runs until you stop it, re-generating built assets on file changes.
+
+# Once this command is waiting for changes, you will need to run in another terminal `yarn build:workers` to fix the "exports undefined" error on start.
+
+# Terminal A
+yarn build-everything:watch # this process will keep running until you stop it
+
+# Terminal B
+yarn build:workers
+
+# If you change any SASS files while running "yarn build-everything:watch" it won't be detected. You will need to run the sass build command.
+
+# Terminal B
+yarn sass
 ```
 
 ## Multiple instances
 
 Since there is no registration for Session, you can create as many accounts as you
 can public keys. Each client however has a dedicated storage profile which is determined by the environment and instance variables.
 
-This profile will change [userData](https://electron.atom.io/docs/all/#appgetpathname)
-directory from `%appData%/Session` to `%appData%/Session-{environment}-{instance}`.
+### Profile Paths
+
+- Linux `~/.config/`
+- macOS `~/Library/Application Support/`
+- Windows `%AppData%/`
+
+This user profile folder will change directories from `[PROFILE_PATH]/Session-{environment}` to `[PROFILE_PATH]/Session-{environment}-{instance}`.
 
 There are a few scripts which you can use:
 
-```
-yarn start-prod - Start production but in development mode
-MULTI=1 yarn start-prod - Start another instance of production
+```shell
+yarn start-prod # Start production but in development mode
+MULTI=1 yarn start-prod # Start another instance of production
 ```
 
-For more than 2 clients, you may run the above command with `NODE_APP_INSTANCE` set before them.
-For example, running:
+For more than 2 clients, you may run the above command with `NODE_APP_INSTANCE` set before them. For example, running:
 
-```
+```shell
 NODE_APP_INSTANCE=alice yarn start-prod
```

Comment:
this should just be MULTI, not NODE_APP_INSTANCE (as you have added on line 244

Reply:
Fixed.